src/imxrt1020/flexio.rs

```diff
@@ -0,0 +1,42 @@
+//! FlexIO pin configurations
+
+use super::pads::{gpio_ad_b1::*, gpio_emc::*};
+use crate::{flexio::Pin, Daisy};
+
+//
+// FLEXIO1
+//
+flexio!(module: 1, offset: 0, pad: GPIO_AD_B1_15, alt: 4, daisy: None);
+flexio!(module: 1, offset: 1, pad: GPIO_AD_B1_14, alt: 4, daisy: None);
+flexio!(module: 1, offset: 2, pad: GPIO_AD_B1_13, alt: 4, daisy: None);
+flexio!(module: 1, offset: 3, pad: GPIO_AD_B1_12, alt: 4, daisy: None);
+flexio!(module: 1, offset: 4, pad: GPIO_AD_B1_11, alt: 4, daisy: None);
+flexio!(module: 1, offset: 5, pad: GPIO_AD_B1_10, alt: 4, daisy: None);
+flexio!(module: 1, offset: 6, pad: GPIO_AD_B1_09, alt: 4, daisy: None);
+flexio!(module: 1, offset: 7, pad: GPIO_AD_B1_08, alt: 4, daisy: None);
+flexio!(module: 1, offset: 8, pad: GPIO_AD_B1_07, alt: 4, daisy: None);
+flexio!(module: 1, offset: 9, pad: GPIO_AD_B1_06, alt: 4, daisy: None);
+flexio!(module: 1, offset: 10, pad: GPIO_AD_B1_05, alt: 4, daisy: None);
+flexio!(module: 1, offset: 11, pad: GPIO_AD_B1_04, alt: 4, daisy: None);
+flexio!(module: 1, offset: 12, pad: GPIO_AD_B1_03, alt: 4, daisy: None);
+flexio!(module: 1, offset: 13, pad: GPIO_AD_B1_02, alt: 4, daisy: None);
+flexio!(module: 1, offset: 14, pad: GPIO_AD_B1_01, alt: 4, daisy: None);
+flexio!(module: 1, offset: 15, pad: GPIO_AD_B1_00, alt: 4, daisy: None);
+flexio!(module: 1, offset: 16, pad: GPIO_EMC_04, alt: 4, daisy: None);
+flexio!(module: 1, offset: 17, pad: GPIO_EMC_05, alt: 4, daisy: None);
+flexio!(module: 1, offset: 18, pad: GPIO_EMC_06, alt: 4, daisy: None);
+flexio!(module: 1, offset: 19, pad: GPIO_EMC_07, alt: 4, daisy: None);
+flexio!(module: 1, offset: 20, pad: GPIO_EMC_08, alt: 4, daisy: None);
+flexio!(module: 1, offset: 21, pad: GPIO_EMC_09, alt: 4, daisy: None);
+flexio!(module: 1, offset: 22, pad: GPIO_EMC_18, alt: 4, daisy: None);
+flexio!(module: 1, offset: 23, pad: GPIO_EMC_19, alt: 4, daisy: None);
+flexio!(module: 1, offset: 24, pad: GPIO_EMC_20, alt: 4, daisy: None);
+flexio!(module: 1, offset: 25, pad: GPIO_EMC_21, alt: 4, daisy: None);
+flexio!(module: 1, offset: 26, pad: GPIO_EMC_22, alt: 4, daisy: None);
+flexio!(module: 1, offset: 27, pad: GPIO_EMC_23, alt: 4, daisy: None);
+flexio!(module: 1, offset: 28, pad: GPIO_EMC_24, alt: 4, daisy: None);
+flexio!(module: 1, offset: 29, pad: GPIO_EMC_25, alt: 4, daisy: None);
+flexio!(module: 1, offset: 30, pad: GPIO_EMC_26, alt: 4, daisy: None);
+flexio!(module: 1, offset: 31, pad: GPIO_EMC_27, alt: 4, daisy: None);
```

Comment:
Same question:

> Mind if I change this 1 to a 0?

I want to make sure this wouldn't affect any of your 1020 testing.

Reply:
This is exactly the issue I stumbled upon when trying to adapt https://github.com/Finomnis/ws2812-flexio to `imxrt1011`, so setting module to 0 seems correct to me.